ssl/ssl_test.cc

```diff
@@ -1034,6 +1038,15 @@ TEST(SSLTest, CipherProperties) {
           NID_auth_rsa,
           NID_md5_sha1,
       },
+      {
```

Comment:
Upstream removed the two ciphers in [this review](https://boringssl-review.googlesource.com/c/boringssl/+/27944/). The removed diffs of `ssl_test.cc` include some tests we may be interested in.

Reply:
thanks Bryce, I added two rules based on the ones removed in the quoted upstream change.